praw/objector.py

```diff
@@ -66,6 +85,35 @@ def objectify(self, data):
                 .issubset(set(data.keys()))):
             parser = self.parsers['ModmailConversation']
             return parser.parse(data, self._reddit)
+        elif isinstance(data, dict) and 'actionTypeId' in data.keys():
```

Comment:
Is there any reason to do `foo.issubset(set(data.keys()))` rather than simply `foo.issubset(data)`?